packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -47,11 +47,13 @@ class UrlLauncherPlugin extends UrlLauncherPlatform {
   /// Registers this class as the default instance of [UrlLauncherPlatform].
   static void registerWith(Registrar registrar) {
     UrlLauncherPlatform.instance = UrlLauncherPlugin();
-    linkDelegate = (LinkInfo linkInfo) => WebLinkDelegate(linkInfo);
     // ignore: undefined_prefixed_name
-    ui.platformViewRegistry.registerViewFactory(viewType, linkViewFactory);
+    ui.platformViewRegistry.registerViewFactory(linkViewType, linkViewFactory);
   }
 
+  @override
+  LinkDelegate linkDelegate = (LinkInfo linkInfo) => WebLinkDelegate(linkInfo);
```

Comment:
This is a getter in the platform interface; does the compiler let you override a getter with this?

Reply:
I changed it to a getter. But to answer your question, yes, the compiler lets me override a getter with a field. Remember the field is nothing but a getter and setter bundled together. So technically it is a getter.